static/app/views/issueList/groupSearchViewTabs/editableTabTitle.tsx

```diff
@@ -51,26 +56,27 @@ function EditableTabTitle({
 
   useEffect(() => {
     if (isEditing) {
-      setTimeout(() => {
-        inputRef?.current?.focus();
-      }, 0);
+      requestAnimationFrame(() => {
+        inputRef.current?.focus();
+      });
     }
   }, [isEditing, inputRef]);
 
   const handleOnChange = (e: React.ChangeEvent<HTMLInputElement>) => {
     setInputValue(e.target.value);
   };
 
-  return isEditing ? (
-    <StyledGrowingInput
-      value={inputValue}
-      onChange={handleOnChange}
-      onKeyDown={handleOnKeyDown}
-      onBlur={handleOnBlur}
-      ref={inputRef}
-    />
-  ) : (
-    <div style={{height: '20px'}}> {label}</div>
+  return (
+    <FocusScope>
+      <StyledGrowingInput
+        value={inputValue}
+        onChange={handleOnChange}
+        onKeyDown={handleOnKeyDown}
+        onBlur={handleOnBlur}
+        ref={inputRef}
+        style={{fontWeight: isSelected ? theme.fontWeightBold : theme.fontWeightNormal}}
```

Comment:
This is redundant with the fontweight set by the `<Tab>` component, but it's necessary to attach this style directly to the growing input to trigger the MutationObserver. Adding it to the emotion styles does not work either.